src/antsibull/cli/antsibull_build.py

```diff
@@ -274,6 +288,25 @@ def _normalize_verify_upstream_options(args: argparse.Namespace) -> None:
         args.ignores = LENIENT_FILE_ERROR_IGNORES
 
 
+def _normalize_announcements_options(args: argparse.Namespace) -> None:
+    if args.command not in ("announcements",):
+        return
+    directory: Path = args.output_dir
+    directory.mkdir(parents=True, exist_ok=True)
+
+
+def _normalize_send_announcements_options(args: argparse.Namespace) -> None:
+    if args.command not in ("send-announcements",):
```

Comment:
Uncomplicate this?
```suggestion
    if args.command != "send-announcements":
```

Reply:
This is how all the `_normalize_foo_options` functions are. Felix says her prefers it this way so it's easier to add extra commands to validate.